packages/cli-lib/sandboxes.js

```diff
@@ -12,7 +11,7 @@ async function createSandbox(accountId, name) {
   try {
     resp = await _createSandbox(accountId, name);
   } catch (err) {
-    logger.error(err.error.message);
+    throw err.error;
```

Comment:
I removed the logger here since it was double printing error messages with the throw here. Didn't see the logger doing anything other than console logging, but let me know if you'd like that back in there

Reply:
Is your intent to exit the process when this happens? If so, you can exit more gracefully via `process.exit(EXIT_CODES.ERROR)`

Reply:
Got it - thanks for that suggestion! Added back the logger and replaced the throw with the process.exit